rmgpy/data/kinetics/kineticsTest.py

```diff
@@ -1091,3 +1091,97 @@ def test_add_atom_labels_for_reaction_3(self):
         self.assertIn('*2',found_labels)
         self.assertIn('*3',found_labels)
 
+    def test_species_preserved_after_generate_reactions(self):
+        """Test that Species objects do not retain changes after generating reactions"""
+        r1 = Species(index=1, label='ethyl', SMILES='C[CH2]')
```

Comment:
Can you use something more descriptive than `r1`?  We more commonly use `r` for reaction than reactant, which can result in some confusion here.

Reply:
I feel like it should be clear from context, but I changed them anyway.